docs/framework/whats-new/index.md

```diff
@@ -28,10 +28,10 @@ manager: "wpickett"
  [.NET Framework 4.5.1](#v451)   
  [.NET Framework 4.5](#core)  
   
- This article does not provide comprehensive information about each new feature and is subject to change. For general information about the .NET Framework, see [Getting Started](http://msdn.microsoft.com/library/c693fd34-88fe-4d90-b332-19eeadf3b7e7). For supported platforms, see [System Requirements](~/docs/framework/get-started/system-requirements.md). For download links and installation instructions, see [Installation Guide](../../../docs/framework/install/guide-for-developers.md).  
+ This article does not provide comprehensive information about each new feature and is subject to change. For general information about the .NET Framework, see [Getting Started](../../../docs/framework/get-started/index.md). For supported platforms, see [System Requirements](~/docs/framework/get-started/system-requirements.md). For download links and installation instructions, see [Installation Guide](../../../docs/framework/install/guide-for-developers.md).  
```

Comment:
this is a migration behavior. we can decide which one to use and change links for consistency later.